web/modules/custom/farm_nfa/src/BreadcrumbBuilder.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Drupal\farm_nfa;
+
+use Drupal\Core\Breadcrumb\Breadcrumb;
+use Drupal\Core\Breadcrumb\BreadcrumbBuilderInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\plan\Entity\PlanInterface;
+
+/**
+ * Breadcrumb builder class for generating breadcrumbs.
+ *
+ * Builds breadcrumbs within the context of the abv_app Drupal package.
+ *
+ * @package Drupal\abv_app
```

Comment:
remove @package and change to comment to, for example. Builds the breadcrumbs for assets and plans.